internal/kafka/internal/config/dataplane_cluster_config.go

```diff
@@ -114,7 +115,8 @@ func NewDataplaneClusterConfig() *DataplaneClusterConfig {
 		EnableReadyDataPlaneClustersReconcile:       true,
 		EnableKafkaSreIdentityProviderConfiguration: true,
 		EnableDynamicScaleUpManagerScaleUpTrigger:   true,
-		Kubeconfig:                                  getDefaultKubeconfig(),
+		EnableDynamicScaleUpManagerScaleDownTrigger: true,
```

Comment:
I know it's still a draft but pointing it just in case: The variable should be named `EnableDynamicScaleDownManagerScaleDownTrigger` in case it ends up here

Reply:
updated